DQM/CTPPS/plugins/CTPPSCommonDQMSource.cc

```diff
@@ -143,16 +143,18 @@ void CTPPSCommonDQMSource::GlobalPlots::Init(DQMStore::IBooker &ibooker) {
     ya->SetBinLabel(6, "45, 220, FR-HR");
     ya->SetBinLabel(7, "45, 220, FR-TP");
     ya->SetBinLabel(8, "45, 220, NR-BP");
-    ya->SetBinLabel(9, "45, 220, NR-TP");
-    ya->SetBinLabel(10, "56, 210, FR-BT");
-    ya->SetBinLabel(11, "56, 210, FR-HR");
-    ya->SetBinLabel(12, "56, 210, FR-TP");
-    ya->SetBinLabel(13, "56, 220, C1");
-    ya->SetBinLabel(14, "56, 220, FR-BT");
-    ya->SetBinLabel(15, "56, 220, FR-HR");
-    ya->SetBinLabel(16, "56, 220, FR-TP");
-    ya->SetBinLabel(17, "56, 220, NR-BP");
-    ya->SetBinLabel(18, "56, 220, NR-TP");
+    ya->SetBinLabel(9, "45, 220, NR-HR");
+    ya->SetBinLabel(10, "45, 220, NR-TP");
+    ya->SetBinLabel(11, "56, 210, FR-BT");
+    ya->SetBinLabel(12, "56, 210, FR-HR");
+    ya->SetBinLabel(13, "56, 210, FR-TP");
+    ya->SetBinLabel(14, "56, 220, C1");
+    ya->SetBinLabel(15, "56, 220, FR-BT");
+    ya->SetBinLabel(16, "56, 220, FR-HR");
+    ya->SetBinLabel(17, "56, 220, FR-TP");
+    ya->SetBinLabel(18, "56, 220, NR-BP");
```

Comment:
I'm just noticing that elsewhere in this PR these are called "NR-BT" (here "NR-BP")...

Reply:
Yes thanks, this is typo (it stays for near-bottom") @grzanka can you fix it?

Reply:
Dear all... sorry. It is a typo in the histogram label. It can be changed to NR-BT (also in 45). Dip publications are all OK in respect to their names. 

<img width="660" alt="Screenshot 2023-05-08 at 16 45 17" src="https://user-images.githubusercontent.com/4973821/236856438-bf38538d-294b-4a1a-a20c-1ca112a4a09b.png">

Reply:
@perrotta thanks for noticing, I can make a fix and submit subsequent commit.

Reply:
First fix ready in https://github.com/cms-sw/cmssw/pull/41518

Reply:
And finally in this PR